test/dtypes/test_affine_quantized.py

```diff
@@ -51,6 +55,44 @@ def test_weights_only(self):
                 else:
                     _ = torch.load(f, weights_only=False)
 
+    @unittest.skipIf(not torch.cuda.is_available(), "Need CUDA available")
+    @unittest.skipIf(not TORCH_VERSION_AFTER_2_3, "sub byte dtype requires torch 2.3+")
+    def test_uintx_target_dtype(self):
+        from torchao.quantization.quant_api import uintx_weight_only
+        for dtype in _DTYPE_TO_BIT_WIDTH.keys():
+            l = torch.nn.Linear(128, 256, dtype=torch.bfloat16, device="cuda")
+            # make sure it runs
+            uintx_weight_only(dtype)(l)
+            l = torch.compile(l)
+            l(torch.randn(1, 128, dtype=torch.bfloat16, device="cuda"))
+
+
+    @unittest.skipIf(not torch.cuda.is_available(), "Need CUDA available")
+    @unittest.skipIf(not TORCH_VERSION_AFTER_2_3, "sub byte dtype requires torch 2.3+")
+    def test_uintx_model_size(self):
```

Comment:
This test seems like it belongs in `test_uintx.py` as a better assert for `test_uintx_weight_only_model_quant`.

Reply:
OK, I can move these